src/main/scala/mesosphere/marathon/raml/TaskConversion.scala

```diff
@@ -21,9 +21,18 @@ object TaskConversion extends HealthConversion with DefaultConversions {
       reservation.volumeIds.toRaml
     }
 
+    val checkStatus: Option[CheckStatus] = task.status.mesosStatus.collect {
+      case mesosStatus if mesosStatus.hasCheckStatus => mesosStatus.getCheckStatus
+    }.map { check =>
+      check.toRaml
+    }.filter { ramlCheck =>
+      ramlCheck.command.nonEmpty || ramlCheck.tcp.nonEmpty || ramlCheck.http.nonEmpty
```

Comment:
why do we need this filter? 🤔

Reply:
The filter removes the check if all the checkresults are empty.   So if there is no value from any check type... do NOT display check result.